util/src/lib.rs

```diff
@@ -15,23 +15,6 @@
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
 #![warn(missing_docs)]
-#![cfg_attr(feature="dev", feature(plugin))]
-#![cfg_attr(feature="dev", plugin(clippy))]
-
-// Clippy settings
-// Most of the time much more readable
-#![cfg_attr(feature="dev", allow(needless_range_loop))]
```

Comment:
if we want to run clippy as a plugin, we should not remove those, but only rename to:

```
#[cfg_attr(feature="cargo-clippy", allow(needless_range_loop))]
```